dashboard/app/models/ability.rb

```diff
@@ -70,7 +70,9 @@ def initialize(user)
       can :manage, Pd::Enrollment, user_id: user.id
       can :workshops_user_enrolled_in, Pd::Workshop
       can :index, Section, user_id: user.id
-      can :membership, Section, followers: {student_user_id: user.id}
+      can :membership, Section, Section.with_student(user) do |section|
+        section
+      end
```

Comment:
I realize that this block looks weird, so here is some context:

With just `can :membership, Section, Section.with_student(user)`, you get an error similar to [this issue](https://github.com/ryanb/cancan/issues/1023) where the controller complains that `The can? and cannot? call cannot be used with a raw sql 'can' definition`. However, if you complete the block, however useless it looks, it will load properly in the controller and does not change the resulting query created by CanCan.